src/web/components/Card/Card.Designs.stories.tsx

```diff
@@ -24,6 +24,16 @@ const Review = Format(
 	'Review',
 );
 
+const ReviewWithStars = Format(
+	{
+		display: Display.Standard,
+		theme: Pillar.News,
+		design: Design.Review,
+	},
+	'Review',
+	0, // star rating
+);
+
```

Comment:
Semantically this story shouldn't live here. I was in a meeting with Source recently and the fact that this list of stories did not align with `Format` was the cause for some confusion so I [updated them](https://github.com/guardian/dotcom-rendering/pull/3131) to fix this - but this PR reimplements the diversion 😭

Reply:
I see what you mean, I had a similar thought but landed on the slightly more relaxed approach anyway, but then I didn't know that it had previously caused confusion outside the team. I'll rejig things.